src/Castle.Facilities.AspNet.Mvc/AspNetMvcControllerFactory.cs

```diff
@@ -100,7 +101,7 @@ private Type FindControllerType(string controllerName)
 		{
 			foreach (var type in this.controllerCandidateTypes)
 			{
-				if (type.Name == controllerName)
+				if (type.Name.EqualsText(controllerName))
```

Comment:
I'd just use `type.Name.Equals(controllerName, StringComparison.OrdinalIgnoreCase)`. That extension method seems a little unnecessary to even exist, and shouldn't be public.

Reply:
Fixed